Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/system/classes/EditClassLayout.java

```diff
@@ -131,14 +133,14 @@ public void execute(Boolean value) {
      *
      * @param name Class name
      * @param groups Class groups
-     * @param engine Class engine
+     * @param engines Class engines
      */
-    public void setClass(String name, String groups, String engine) {
+    public void setClass(String name, String groups, String engines) {
 
         if (name != null) {
             this.nameField.setValue(name);
             this.nameField.setDisabled(true);
-            this.enginesPickList.setValue(engine);
+            this.enginesPickList.setValues(engines.split(", "));
```

Comment:
Not sure... what happens if there is not space character before or after the comma? Does it still match? I would split by ",", and then trim the result, e.g. as follows:
```Java
Arrays.asList(engines.split(",")).stream().map(String::trim)
```

(Stream was introduced in Java 8)